domain/src/memory.js

```diff
@@ -86,6 +88,17 @@ function selectNotDeleted(dict) {
 }
 
 /**
+ * Полностью обновляет предмет(кроме id)
+ * @returns {Promise.<Number, Error>} Количество найденных объектов
+ */
+async function updateItemFully(db, coll, id, item) {
```

Comment:
I named this function "fully", because it fully update item, in fact replace item